lcc_lokavaluto_app_connection/models/company.py

```
@@ -9,3 +11,57 @@ class Company(models.Model):
     monujo_ios_app_url = fields.Char(string="Monujo iOS app URL")
     activate_automatic_topup = fields.Boolean("Activate Automatic Topup")
     commission_product_id = fields.Many2one("product.product", string="Commission Product")
+
+
+    def _cron_import_new_digital_currency_debit_requests(self):
+        """Create the debit requests in Odoo for all the debit
+        transactions performed since the last check."""
+
+        # Extract last transactions received in the Safe Wallet
+        transactions = self._retrieve_last_debit_transactions()
+
+        # For each transaction received:
+        # - retrieve the transaction id
+        # - check if any debit request exists with this transaction id
+        # - if yes, do nothing
+        # - if no, create a new debit request
+        for transaction in transactions:
+            tx_id = transaction.get("tx_id", "")
```

hmm, why silence this error in the data ? A log at least should be important. And we can probably `continue` also.

---

add a check to directly continue before performing the search on debit.requests in the following line.
add warning log too.